### PR TITLE
[AP] Create 2.9.0 docs

### PR DESCRIPTION
**Pull request by philipliu**
This was generated using `VERSION=2.9.0 yarn ap:versions:new`.

### CODE REVIEW

**Comment by ElliotFriend** [comment]
I'm honestly a little surprised and delighted that the command (looks to have) worked out reasonably well 🤣

File: openapi/anchor-platform/versions/callbacks-2.9.0.yaml [hunk 1]
```diff
@@ -0,0 +1,1309 @@
+openapi: 3.1.0
+info:
+  version: 3.0.0
```
Comment (ElliotFriend) [inline openapi/anchor-platform/versions/callbacks-2.9.0.yaml:3]:
```suggestion
  version: 2.9.0
```

When I was working on #909, I changed the `info.version` in the specfiles to `3.0.0`, 'cause I thought that would be next up. You'll probably want to change that in these files. Maybe even in the `/openapi/anchor-platform/main-*.yaml` files, too, depending on what your release cycles look like

Comment (philipliu) [reply openapi/anchor-platform/versions/callbacks-2.9.0.yaml:3]: Thanks for the catch! `2.9` should be the last minor version before our `3.0` release, so I think it makes sense to only update the versions in `/openapi/anchor-platform/*-2.9.0.yaml`s.

Comment (lijamie98) [reply openapi/anchor-platform/versions/callbacks-2.9.0.yaml]: Is there a way to know what's changed from `2.8.x` to `2.9.0`?

**Comment by Ifropc** [comment]
Do we want to create docs for each new minor version? Or will we skip if there are no changes in docs? 🤔

File: ap_versioned_docs/version-2.9.0/admin-guide/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+## Installation
+
+import { CodeExample } from "@site/src/components/CodeExample";
+
+The easiest way to install the Anchor Platform is to pull the [docker image][anchor-platform-image].
+
+<CodeExample>
+
+```bash
+docker pull stellar/anchor-platform:latest
```
Comment (Ifropc) [inline ap_versioned_docs/version-2.9.0/admin-guide/getting-started.mdx:15]: We still pill `latest` here, I think we should start pinning versions from now on. Maybe do replacement of 
`stellar/anchor-platform:latest` -> `docker pull stellar/anchor-platform:<vesion>` as part of `yarn ap:versions:new` in the future? (It's a nice to have but don't think is mandatory)

Comment (philipliu) [reply ap_versioned_docs/version-2.9.0/admin-guide/getting-started.mdx:15]: Good call. I'll rename it manually for now.

**Comment by philipliu** [comment]
> Do we want to create docs for each new minor version? Or will we skip if there are no changes in docs? 🤔

I think you mean the patch version. Yeah, I think it makes sense to publish docs for each minor version instead. i.e. have docs for `2.8`, `2.9`, and 3.0`, etc.

**Pull request merged**